concepts/booleans/introduction.md

```diff
@@ -3,3 +3,14 @@
 Booleans in Java are represented by the `boolean` type, which values can be either `true` or `false`.
 
 Java supports three boolean operators: `!` (NOT), `&&` (AND), and `||` (OR).
+
+! negates the boolean, && takes two booleans and results in true if they're both true. || results in true if any of the two booleans is true.
```

Comment:
maybe we can merge this new line with the previous one.
Something like:

Java supports three boolean operators: 
* `!` (NOT): it negates the boolean

and so on

Reply:
Just made that adjustment.